docs/conf.py

```diff
@@ -127,85 +128,22 @@
 html_static_path = ["_static"]
 # html_css_files = ["stravalib.css"]
 
-# Short title for the navigation bar.
-html_short_title = "Stravalib Python Package Documentation"
-
 # Instagram always throws 429 so ignore it
 linkcheck_ignore = [r"https://www.instagram.com/accounts/login/"]
-# The name of an image file (relative to this directory) to place at the top
-# of the sidebar.
-# html_logo = None
-
-# The name of an image file (within the static path) to use as favicon of the
-# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-# html_favicon = None
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
 
-# If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
```

Comment:
ALl of this config is dated. we do all of our config now via pydata_sphinx_theme above via a dict.